tests/saputils_test.py

```diff
@@ -0,0 +1,113 @@
+"""
+Unitary tests for saputils.py.
+
+:author: sisingh
+:organization: SUSE LLC
+:contact: sisingh@suse.com
+
+:since: 2020-03-26
+"""
+
+# pylint:disable=C0103,C0111,W0212,W0611
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import logging
+import unittest
+import filecmp
```

Comment:
@Simranpal We don't need to import `filecmp` and `shutil`. Please remove them

Reply:
Thanks, done!